app/src/main/java/org/mozilla/fenix/home/topsites/TopSiteItemViewHolder.kt

```diff
@@ -92,7 +97,24 @@ class TopSiteItemViewHolder(
                         }
                     }
 
-                    binding.topSiteCard.setCardBackgroundColor(backgroundColor)
+                    binding.faviconCard.setCardBackgroundColor(backgroundColor)
+
+                    val textColor = currentState.currentWallpaper.textColor
+                    if (textColor != null) {
+                        val color = Color(textColor).toArgb()
+                        val colorList = ColorStateList.valueOf(color)
+                        binding.topSiteTitle.setTextColor(color)
+                        TextViewCompat.setCompoundDrawableTintList(binding.topSiteTitle, colorList)
```

Comment:
Let's move this line above or below the other `setTextColor` so that we could group the common operations right after another.